services/movie_session.py

```diff
@@ -21,14 +25,14 @@
 
 
 def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
-    return MovieSession.objects.get(id=movie_session_id)
+    return get_object_or_404(MovieSession, id=movie_session_id)
 
 
 def update_movie_session(
-    session_id: int,
-    show_time: str = None,
-    movie_id: int = None,
-    cinema_hall_id: int = None,
+        session_id: int,
+        show_time: str = None,
```

Comment:
Consider using a `datetime` object for `show_time` instead of a string to ensure proper handling and validation of date and time.